content-src/components/DiscoveryStreamComponents/DSContextFooter/DSContextFooter.jsx

```diff
@@ -9,13 +9,13 @@ import React from "react";
 // Animation time is mirrored in DSContextFooter.scss
 const ANIMATION_DURATION = 3000;
 
-const StatusMessage = ({ icon, fluentID }) => (
+export const StatusMessage = ({ icon, fluentID }) => (
   <div className="status-message">
     <span
       aria-haspopup="true"
       className={`story-badge-icon icon icon-${icon}`}
     />
-    <div className="story-context-label" data-l10n-id={fluentID} />
+    <div className="story-context-label clamp" data-l10n-id={fluentID} />
```

Comment:
Probably safe to remove clamp here, to make it work we need -webkit-line-clamp same as below